coda/coda_validate/views.py

```diff
@@ -28,7 +28,6 @@
 
 
 XML_HEADER = "<?xml version=\"1.0\"?>\n%s"
-DOMAIN = Site.objects.get_current().domain
```

Comment:
This needed to be moved into the (only) class that used it which is `CorrectMimeTypeFeed. It was causing an exception to be raised when the trying to run the migrations.